src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentNotebookRequest.cs

```diff
@@ -256,4 +256,32 @@ public static readonly
             RequestType<UpdateAgentNotebookRunPinParams, ResultStatus> Type =
             RequestType<UpdateAgentNotebookRunPinParams, ResultStatus>.Create("agent/updatenotebookpin");
     }
+
+     /// <summary>
+    /// SQL Agent Notebook materialized params
+    /// </summary>
+    public class DeleteNotebookMaterializedParams : TaskRequestDetails
+    {
+        public string OwnerUri { get; set; }
+        public string TargetDatabase { get; set; }
+        public AgentNotebookHistoryInfo agentNotebookHistory { get; set; }
+    }
+
+    /// <summary>
+    /// SQL Agent Notebook materialized result
+    /// </summary>
+    public class DeleteNotebookMaterializedResult : ResultStatus
+    {
+        public string NotebookMaterialized { get; set; }
```

Comment:
>NotebookMaterialized [](start = 22, length = 20)

Is this supposed to be the name of the notebook that was deleted? Suggest having the name be MaterializedNotebook then - right now this sounds like it's telling us whether the notebook was materialized (which makes no sense in this context)